examples/06_benchmarks/movielens.ipynb

```diff
@@ -72,28 +72,39 @@
    "execution_count": 1,
```

Comment:
<p>mmm it is taking much longer to train. Mostly due to lightGCN. Sahiti, do you know what is happening?</p><p> </p><p>FYI @simonzhaoms @chuyangke @fazamani</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/recommenders/pull/1758/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>